src/bin/pg_autoctl/monitor.c

```diff
@@ -623,13 +865,15 @@ printCurrentState(void *ctx, PGresult *result)
 		}
 	}
 
-	fprintf(stdout, "%*s | %6s | %5s | %5s | %17s | %17s\n",
+	fprintf(stdout, "%*s | %6s | %5s | %5s | %17s | %17s | %18s | %18s\n",
 			maxNodeNameSize, "Name", "Port",
-			"Group", "Node", "Current State", "Assigned State");
+			"Group", "Node", "Current State", "Assigned State",
+			"Candidate Priority", "Replication Quorum");
```

Comment:
Wow, +18 columns is a lot. Let's make it "prio" and "quorum" maybe? we're not going to fit in most screen sizes. Maybe it's time to have a verbose and a non-verbose output. In another branch I have implement `--json` for the output here, too.

Let's have a non-verbose default output with only name, port, current state, assigned state, priority, quorum; and a verbose one where we add in the group and node IDs. I don't think those are needed for normal users, now that I think about it.